docs/hpc/systems/supported/catscan.md

```diff
@@ -0,0 +1,113 @@
+Maximum memory pitch:                          2147483647 bytes 
+Texture alignment:                             512 bytes 
+Concurrent copy and kernel execution:          Yes with 2 copy engine(s) 
+Run time limit on kernels:                     No 
+Integrated GPU sharing Host Memory:            No 
+Support host page-locked memory mapping:       Yes 
+Alignment requirement for Surfaces:            Yes 
+Device has ECC support:                        Disabled 
+Device supports Unified Addressing (UVA):      Yes 
+Device supports Compute Preemption:            Yes 
+Supports Cooperative Kernel Launch:            Yes 
+Supports MultiDevice Co-op Kernel Launch:      Yes 
+Device PCI Domain ID / Bus ID / location ID:   0 / 94 / 0 
+Compute Mode: < Default (multiple host threads can use ::cudaSetDevice() with device simultaneously) >
+```
+
+## Queue Configuration
+
+At the moment, there is no resource manager/scheduler. Depending on how the resources are used, this may change.
+
+## Additional Notes
+
+### Authentication:
+
+Currently authentication is LDAP -- the same credentials that you would use to access gmail.lbl.gov. We are in the process of evaluating OTP over ssh keys, or simply OTP.
+
+### Accessing the Compute nodes:
+
+You will need to generate ssh-keys for intra-cluster access to the compute nodes. 
+
+To do this run: 
+
+```jsx
+ssh-keygen -t ed25519
+```
+
+Then run 
+
+```jsx
+cat ~/.ssh/id_ed25519.pub >> ~/.ssh/authorized_keys
+```
+
+Then finally
+
+```jsx
+chmod 600 ~/.ssh/authorized_keys
+```
+
+The suggested default name and location should be fine. You will be prompted for a password, but to leave it blank just hit enter. Passwords can interfere with intra-cluster node communication when launching jobs, particularly with a scheduler should we choose to deploy one.
+
+## Storage for Data:
+
+Each user will be granted space under /clusterfs/bebb/users for data. There is also a group directory to which the group has write permission. This directory should be used for custom builds of software of which the group may want to take advantage. This is modeled after what we offer on our clusters in 1275, but this is your cluster so you can choose to use it how you desire.
+
+## Software Module Farm:
+
+See [Documentation on using "modules"](/hpc/software/module-management/).
+
+### Apptainer (formerly known as Singularity):
+
+Some of you have expressed the need to import custom software built on different architectures. Instead of re-inventing this on the catscan architecture, you can use Apptainer. 
+
+Apptainer enables users to have full control of their environment. Apptainer can be used to package entire scientific workflows, software and libraries, and even data. 
+
+To get started, check out these links:
+
+[Documentation on using Singularity on a cluster called Savio that we manage on UCB cluster](http://research-it.berkeley.edu/services/high-performance-computing/using-singularity-savio)
```

Comment:
```suggestion
[Documentation on using Singularity on a cluster called Savio that we manage on UCB cluster](http://research-it.berkeley.edu/services/high-performance-computing/using-singularity-savio){:target="_blank"} {{ ext }}
```

Reply:
It's not clear to me why savio-singularity is brought up here.